### PR TITLE
Fix linter

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-catalog-diff-viewer.svg)](https://github.com/voxpupuli/puppet-catalog-diff-viewer/blob/master/LICENSE)
+[![CI](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml/badge.svg)](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml)
+[![Donated by biemond](https://img.shields.io/badge/donated%20by-biemond-fb7047.svg)](#transfer-notice)
+<!-- update after forge release -->
+![Puppet Forge Endorsement](https://img.shields.io/puppetforge/e/biemond/wildfly.svg)
```
Comment (bastelfreak) [inline README.md:7]: I suggest to already update them. Otherwise there will be wrong links in the release itself.

**Comment by rwaffen** [comment]
omg, okay, should have look at the matrix first, its all EOL versions XD

**Comment by rwaffen** [comment]
but just wanna fix linters here, after this, i go on and do the other things, so please ignore ci ftm

File: README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-catalog-diff-viewer.svg)](https://github.com/voxpupuli/puppet-catalog-diff-viewer/blob/master/LICENSE)
+[![CI](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml/badge.svg)](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml)
+[![Donated by biemond](https://img.shields.io/badge/donated%20by-biemond-fb7047.svg)](#transfer-notice)
+
+![Puppet Forge Endorsement](https://img.shields.io/puppetforge/e/voxpupuli/wildfly)
+![Puppet Forge Version](https://img.shields.io/puppetforge/v/voxpupuli/wildfly)
+![Puppet Forge Score](https://img.shields.io/puppetforge/f/voxpupuli/wildfly)
+![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/voxpupuli/wildfly)
```
Comment (bastelfreak) [inline README.md:10]: you can also add:
```
[![puppetmodule.info docs](https://www.puppetmodule.info/images/badge.png)](https://www.puppetmodule.info/m/puppet-wildfly)
```

Comment (rwaffen) [reply README.md:10]: will do

Comment (rwaffen) [reply README.md:10]: ah thats this one, i removed it, because it looked broken

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**